project-base/app/translations/dataFixtures.cs.po

```diff
@@ -307,6 +307,9 @@ msgstr "<b>Zvolili jste platbu kreditní kartou. Prosím proveďte ji do dvou pr
 msgid "<div class=\"gjs-text-ckeditor\"> description - Lorem ipsum dolor sit amet, </div> %productsFirstRow% <div class=\"gjs-text-ckeditor\"> consectetur </div> %productsSecondRow% <div class=\"gjs-text-ckeditor\">adipiscing elit. Vivamus felis nisi, tincidunt sollicitudin augue eu, laoreet blandit sem. Donec rutrum augue a elit imperdiet, eu vehicula tortor porta. Vivamus pulvinar sem non auctor dictum. Morbi eleifend semper enim, eu faucibus tortor posuere vitae. Donec tincidunt ipsum ullamcorper nisi accumsan tincidunt. Aenean sed velit massa. Nullam interdum eget est ut convallis. Vestibulum et mauris condimentum, rutrum sem congue, suscipit arcu. \\nSed tristique vehicula ipsum, ut vulputate tortor feugiat eu. Vivamus convallis quam vulputate faucibus facilisis. Curabitur tincidunt pulvinar leo, eu dapibus augue lacinia a. Fusce sed tincidunt nunc. Morbi a nisi a odio pharetra laoreet nec eget quam. In in nisl tortor. Ut fringilla vitae lectus eu venenatis. Nullam interdum sed odio a posuere. Fusce pellentesque dui vel tortor blandit, a dictum nunc congue. </div>"
 msgstr "<div class=\"gjs-text-ckeditor\"> description - Lorem ipsum dolor sit amet, </div> %productsFirstRow% <div class=\"gjs-text-ckeditor\"> consectetur </div> %productsSecondRow% <div class=\"gjs-text-ckeditor\">adipiscing elit. Vivamus felis nisi, tincidunt sollicitudin augue eu, laoreet blandit sem. Donec rutrum augue a elit imperdiet, eu vehicula tortor porta. Vivamus pulvinar sem non auctor dictum. Morbi eleifend semper enim, eu faucibus tortor posuere vitae. Donec tincidunt ipsum ullamcorper nisi accumsan tincidunt. Aenean sed velit massa. Nullam interdum eget est ut convallis. Vestibulum et mauris condimentum, rutrum sem congue, suscipit arcu. \\nSed tristique vehicula ipsum, ut vulputate tortor feugiat eu. Vivamus convallis quam vulputate faucibus facilisis. Curabitur tincidunt pulvinar leo, eu dapibus augue lacinia a. Fusce sed tincidunt nunc. Morbi a nisi a odio pharetra laoreet nec eget quam. In in nisl tortor. Ut fringilla vitae lectus eu venenatis. Nullam interdum sed odio a posuere. Fusce pellentesque dui vel tortor blandit, a dictum nunc congue. </div>"
 
+msgid "<div class=\"gjs-text-ckeditor\">Article text for search testing, the search phrase is &#34;Dina&#34;.</div>"
```

Comment:
I was surprised I saw this addition and no deletion, I was expecting the original string to be removed from here. I found out the same string is used in `ArticleDataFixture` where the "gjs-text-ckeditor" div is excluded from the message id. 

So I see in `BlogArticleDataFixture`, we include the divs in the message IDs, while in `ArticleDataFixture`, we exclude the divs from the translation message IDs. 

Does it bother me? I do not think so :smile: